mqtt-bridge/src/main/java/io/strimiz/strimzi_mqtt_bridge/core/MqttServerHandler.java

```diff
@@ -0,0 +1,57 @@
+package io.strimiz.strimzi_mqtt_bridge.core;
+
+
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.SimpleChannelInboundHandler;
+import io.netty.handler.codec.mqtt.*;
```

Comment:
As code style in Strimzi we avoid using `*` for importing classes. It's better to list what's used here.